pkgs/top-level/python-packages.nix

```diff
@@ -1749,6 +1749,11 @@ in {
 
   cozy = callPackage ../development/python-modules/cozy { };
 
+  cppe = toPythonModule (pkgs.cppe.override {
+    inherit (self) python pybind11 numpy h5py numba scipy pandas pytest;
```

Comment:
This shouldn't be required. I think we can input python and use with python in the package.